### ISSUE TITLE
Default encoding not working for EDMC < 3.5

### ISSUE THREAD

**Issue by SayakMukhopadhyay** [SayakMukhopadhyay/EDMC-Discord-Presence#16]
Default encoding is giving an error 
```
Traceback (most recent call last):
  File "plug.pyo", line 193, in load_plugins
  File "plug.pyo", line 94, in init
  File "C:\Users\jesme\AppData\Local\EDMarketConnector\plugins\DiscordPresence\load.py", line 151, in <module>
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-10: ordinal not in range(128
)
```

This is probably what @Poneyy was talking about in https://github.com/SayakMukhopadhyay/EDMC-Discord-Presence/pull/15#issuecomment-584565492

**Comment by VAKazakov** [comment]
str.encode() by default uses encoding, taken from sys.getdefaultencoding()

**Comment by SayakMukhopadhyay** [comment]
Fixed by 335af3f

**Comment by derrickmehaffy** [comment]
actually fixed by https://github.com/SayakMukhopadhyay/EDMC-Discord-Presence/commit/20d661f98d6552bfced41aa7867dace3d0ac48ed
